Providers/Profile.swift

```diff
@@ -715,14 +713,18 @@ open class BrowserProfile: Profile {
                         return
                     }
 
-                    let url = "https://mozilla-anonymous-sync-metrics.moo.mx/post/bookmarkvalidation".asURL!
+                    guard let url = URL(string: "https://mozilla-anonymous-sync-metrics.moo.mx/post/bookmarkvalidation") else {
+                        return
+                    }
+
                     var request = URLRequest(url: url)
                     request.httpMethod = "POST"
                     request.httpBody = body
-                    request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+                    request.addValue("application/json", forHTTPHeaderField: "Content-Type")
```

Comment:
Why the change?

Reply:
hm.. likely an oversight.. i copied in the non-alamofire HTTP request code from telemetry-ios